stable-diffusion/anaconda-project.yml

```diff
@@ -0,0 +1,120 @@
+# To reproduce: install 'anaconda-project', then 'anaconda-project run'
+
+# required: name MUST match the directory name.
+#           name must be of the form r'[a-z_]+'
+name: stable-diffusion
+# required: short project description, used in the main gallery
+description: An example panel app for stable diffusion
+
+# required: config 
+examples_config:
+  #### REQUIRED ###
+  # Creation date following the YYYY-MM-DD format
+  created: 2022-01-30
+  # List of the GITHUB handle of the authors/maintainers
+  maintainers:
+  - "sandhujasmine"
+  # List of labels displayed in the project card
+  # Each label must be a name (e.g. panel) that
+  # refers to a SVG badge located in doc/_static/labels
+  # (e.g. doc/_static/labels/panel.svg)
+  labels:
+  - "channel_conda-forge"
+  - "panel"
+
+  ### OPTIONAL ###
+  # Listed deployments will by default automatically be started.
+  # Maximum number of deployments is 2.
+  # Each deployment must declare the command it deploys, options
+  # include "notebook" or "dashboard".
+  deployments:
+    # Will be deployed at {projname_with_hyphens}-notebook.pyviz.demo.anaconda.com
+  - command: notebook
+    # Will be deployed at {projname_with_hyphens}.pyviz.demo.anaconda.com
+  - command: dashboard
+      # [optional] Set the AE5 container resource profile.
+      # Options include: "default", "medium" (default), "large"
+    resource_profile: medium
+      # [optional] Automatically start the deployment on AE5 when a PR
+      # modifying the project is merged. Default is true.
+    auto_deploy: true
+
+  # to build the website (e.g. too long or require too much data).
+  # This indicates the system not to run them.
+  skip_notebooks_evaluation: false
+  # For projects that get their data either by:
+  # - automatically generating it
+  # - providing indications
+  no_data_ingestion: false
+
+# required: (needed internally)
+user_fields: [examples_config]
+
+# required: list of the channels needed to solve the environment
+channels:
+- pyviz
+- defaults
+- conda-forge
```

Comment:
I would guess that depending on `conda-forge` is going to be enough for this project. There are probably some missing packages on `defaults` anyway, and both `conda-forge` and `pyviz` have the latest version of Panel.

Reply:
For dev setup I didn't use defaults. 
I tested this using these channels since I thought we may want to point to main. But ok to remove.